model_builder/model_builder/handlers/util/model_builder_sbml.py

```diff
@@ -23,8 +23,8 @@
 from gillespy2.sbml.SBMLimport import convert
 from gillespy2.stochss.StochSSexport import export
 
-from .stochss_base import StochSSBase
-from .stochss_errors import StochSSFileNotFoundError
+from .model_builder_base import StochSSBase
+from .model_builder_errors import StochSSFileNotFoundError
 
 class StochSSSBMLModel(StochSSBase):
```

Comment:
StochSSSBMLModel => GillesPy3DSBMLModel

Then change all references to "StochSSSBMLModel" in the code

Reply:
Also Change "StochSSBase" class and "StochSSFileNotFoundError"